test/testunusedvar.cpp

```diff
@@ -2024,6 +2025,27 @@ class TestUnusedVar : public TestFixture {
         ASSERT_EQUALS("", errout_str());
     }
 
+    void structmemberStructuredBinding() { // #13107
+        checkStructMemberUsage("struct S { int a, b; };\n"
```

Comment:
can this testcase be reduced.. maybe:

    "struct S { int a, b; };\n"
    "void foo(const S& s) {\n"
    "    auto [x,y] = s;\n"
    "    dostuff(x,y);\n"
    "}\n"